docs/ref/settings.txt

```diff
@@ -2644,6 +2644,13 @@ A ready-to-use instance of the storage backends can be retrieved from
 :data:`django.core.files.storage.storages`. Use a key corresponding to the
 backend definition in :setting:`STORAGES`.
 
+.. note::
+
+   Defining this setting overrides the default value, it's NOT "merged" with
+   the default value. For example, if you redefine the setting and specify
+   only a ``staticfiles`` storage, without ``default`` storages, then users
+   of your site will not be able to upload files.
```

Comment:
I have doubts, we don't have such notes in a similar settings, e.g. `TEMPLATES, `CACHES`, or `DATABASES` and it's not accurate (you can upload files without the `default` storage).

Reply:
Yes, it's true other similar settings don't have such warning, but the ones that have a non-empty value have a single top-level key `default`. AFAIK, this is the only setting with 2 keys in its default value.

Maybe it's just because I'm used to the old settings `DEFAULT_FILE_STORAGE` and `STATICFILES_STORAGES` and in a few months I will have forgotten about it... 

Can you upload files with only `staticfiles` defained?

Reply:
> Can you upload files with only `staticfiles` defained?

You can upload files without a `default` storage, always using custom storages.

Reply:
Ah right, got it: by passing a `storage` to the file field directly. I forgot that custom storage classes don't have to be present in this setting.